commit b7d5c4d0bcb63cce687a7e150e3db38c2ae56563 (refs/changes/01/5201/2)
Author: Trent Mick <trentm@gmail.com>
Date:   2018-12-06T16:05:30-08:00 (10 months ago)
    
    MANTA-4030 many images using node 0.10 or 0.12 broken by transitive lru-cache@4.1.4 dep (pinning lru-cache version is no longer necessary after 4.1.5 release)
    Reviewed by: Josh Wilsdon <josh@wilsdon.ca>
    Approved by: Josh Wilsdon <josh@wilsdon.ca>

diff --git a/package.json b/package.json
index 38ea6a4..2a9565d 100644
--- a/package.json
+++ b/package.json
@@ -1,14 +1,13 @@
 {
     "name": "muskie",
     "description": "Manta public-facing WebAPI",
-    "version": "1.0.1",
+    "version": "1.0.2",
     "author": "Joyent (joyent.com)",
     "private": true,
     "repository": {
         "type": "git",
         "url": "git+ssh://git@github.com:joyent/muskie.git"
     },
-    "// lru-cache": "attempting to pin version while satisfying transitive deps, because lru-cache@4.1.4 includes yallist@3 which breaks node --abort-on-uncaught-exception with node <v4",
     "dependencies": {
         "ajv": "5.2.2",
         "ajv-keywords": "2.1.0",
@@ -30,9 +29,9 @@
         "keyapi": "git+https://github.com/joyent/keyapi.git#e14b3d58",
         "libmanta": "git+https://github.com/joyent/node-libmanta.git#v1.1.1",
         "libuuid": "0.1.2",
-        "lru-cache": "4.1.3",
+        "lru-cache": "4.1.5",
         "lstream": "0.0.4",
-        "mahi": "2.2.2",
+        "mahi": "2.2.3",
         "marlin": "git+https://github.com/joyent/manta-marlin.git#master",
         "mime": "1.2.11",
         "moray": "3.4.1",
